CONTRIBUTING.md

```diff
@@ -33,3 +46,65 @@ Use the review apps to verify callback functionality (note these examples are wr
 `python setup.py sdist`
 
 Note: Distributable file will be located in ./dist
+
+## Making a Contribution
+_For larger features, your contribution will have a higher likelihood of getting merged if you create an issue to discuss the changes that you'd like to make before you create a pull request._
+
+1. Create a pull request and tag the Plotly team (`@plotly/dash`) and tag / request review from [@Marc-Andre-Rivet](https://github.com/Marc-Andre-Rivet) and [@valentijnnieman ](https://github.com/valentijnnieman).
+2. After a review has been done and your changes have been approved, create a prerelease and comment in the PR. Version numbers should follow [semantic versioning][]. To create a prerelease:
+    * Add `-rc1` to `package.json` e.g. `0.14.0-rc1`
+    * Update the `unpkg` link in `./dash_table/__init__.py`, replacing `__version__` with your release candidate (e.g. `"0.14.0-rc1"`)
```

Comment:
I think this is wrong. If you npm run build:js && npm run build:py, shouldn't the __init__.py pick up the new version automatically?